### PR TITLE
Add failureaccess dependency.

### PR DESCRIPTION
**Pull request by uri-canva**
# Checklist

- [ ] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ ] I have received the approval from the maintainers to make this change.
- [ ] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Discussion thread for this change

Issue number: https://github.com/bazelbuild/intellij/issues/1443

# Description of this change

See issue.

### CODE REVIEW

File: intellij_platform_sdk/BUILD [hunk 1]
```diff
@@ -327,6 +327,26 @@ java_library(
...
+java_library(
+    name = "failureaccess_for_external_binaries",
+    visibility = ["//aspect/tools:__pkg__"],
+    exports = select_from_plugin_api_directory(
+        android_studio = [":failureaccess"],
+        clion = [":failureaccess"],
+        intellij = [":failureaccess"],
+    ),
+)
+
```
Comment (uri-canva) [inline intellij_platform_sdk/BUILD]: This probably isn't correct as only the newer versions have this jar, but I'm not sure which ones.

**Comment by kylecordes** [comment]
@uri-canva This failed to build in the CI, and also fails to build locally for me. (I was going to try it with the latest IJ + latest Bazel and report success/failure... didn't get that far.)

**Comment by uri-canva** [comment]
Thanks for letting me know! I changed the PR so it only affects IntelliJ UE 2019.3. It's likely that IntelliJ CE 2019.3 needs this as well, and probably some of the other IDEs, but I don't have them to test that.

**Comment by brendandouglas** [comment]
Thanks, I'll apply this upstream

**Pull request closed without merging**